cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityTopicOperator.java

```diff
@@ -310,8 +310,28 @@ public void setContainerEnvVars(List<ContainerEnvVar> envVars) {
         templateContainerEnvVars = envVars;
     }
 
-
     public void setContainerSecurityContext(SecurityContext securityContext) {
         templateContainerSecurityContext = securityContext;
     }
+
+    /**
+     * Transforms map to log4j properties file format
+     * @param properties map with properties
+     * @return
+     */
+    @Override
+    protected String createPropertiesString(OrderedProperties properties) {
+        properties.addPair("monitorInterval", "15");
```

Comment:
Is this 15 seconds? How do we know that is the right number? Also, since it is used on several places, should we have a constant?

Reply:
It is 15 seconds. I consider it a reasonable value. The `log4j2.properties` file is rewritten after periodic reconciliation.